app.js

```diff
@@ -404,6 +404,15 @@ let DNSResult = create({
 
   render: function() {
     let j = this.props.json;
+
+    if (j.SRVSkipped) {
+      return (
+        <div className="dns">
+          <h2>.well-known information found -&gt; SRV record-check skipped</h2>
```

Comment:
This isn't really accurate.

```suggestion
          <h2>server name/.well-known result contains explicit port number: no SRV lookup done</h2>
```

Reply:
Also: h2 is a bit big: it looks like an error of some kind. Shouldn't this be in a "DNS results" section, with smaller text saying that we skipped SRV lookup?

(The existing `if (j.SRVRecords == null)` logic is wrong too, but there is no need to make the problem worse)